lib/rubocop/cop/layout/indentation_consistency.rb

```diff
@@ -137,15 +137,60 @@ def autocorrect(node)
 
         private
 
+        def access_modifier?(node)
```

Comment:
Don't we have somewhere such a node helper already? Maybe you should grep the rest of the code. Might be named `bare_access_modifier?` on the current `master`.

Reply:
`bare_access_modifier?` exists, and it seems we should use it instead of `access_modifier?`, thanks.

However, it seems we still have to check `node.send_type?` first, because there are some nodes which do not define `bare_access_modifier?`:

```
       undefined method `bare_access_modifier?' for "s(:def, :priv,\n  s(:args), nil)":RuboCop::AST::DefNode
```

I suppose we could promote `bare_access_modifier?` and the half-dozen other methods it needs from `MethodDispatchNode` to `Node`, but I'd worry about cluttering up `Node`.

Please advise.